policy_agent.py

```diff
@@ -134,7 +134,7 @@ def run(self):
             self._start_leader_thread()
 
         # Ensure the tier exists.
-        metadata = {"order": 50}
+        metadata = {"order": 100}
```

Comment:
yes, it should probably be a constant.

I don't have solid mathematical reason for making this 100 as opposed to 1000 or 10000.

Basically this number is the "order" of the tier that all k8s policies are created in.

I believe @fasaxc said that you can have negative orders, so we're technically not limited by this number, but making it a multiple of 10 felt nicer to me than keeping it at 50.  You can still have 100 other tiers from 0 -> 99.